.travis.yml

```diff
@@ -0,0 +1,17 @@
+sudo: required
+
+language: ruby
+
+services:
+  - docker
+
+install: git clone --depth 1 https://github.com/sstephenson/bats.git
```

Comment:
I don't think it's appropriate to submodule bats into this project. People interested in running tests will have to install bats.